docs/source/ledger-nano.rst

```diff
@@ -0,0 +1,261 @@
+      :width: 400px
+
+#. Make sure your Ledger Nano is connected, unlocked, and the Insolar application is launched on it.
+
+   .. _enter_key_number:
+
+#. If required, enter the key number. Ledger Nano can store multiple private keys—each to an individual Insolar Wallet. Every key stored in the device has a number. By default, the number of the first key is ``0``.
+
+   .. important:: Remember the number of this private key. You are required to specify it upon every login to use a particular Insolar MainNet Wallet.
+
+   .. image:: imgs/mig-test/key-number.png
+      :width: 500px
+
+#. Check the boxes to allow anonymous data collection and agree to the terms of use and click :guilabel:`CONNECT TO LEDGER NANO S`.
+
+   .. image:: imgs/mig-test/connect-n.png
+      :width: 500px
+
+#. In the browser's prompt window, select the :guilabel:`Nano S` or :guilabel:`Nano X` device and click :guilabel:`Connect`.
+
+   .. image:: imgs/mig-test/select-n.png
+      :width: 400px
+
+#. In the dashboard of the Ledger Nano device, the Insolar application prompts you to confirm the :guilabel:`Create Account` command.
+   
+   .. image:: imgs/mig-test/ledger-s-create-account.png
+      :width: 300px
+
+   Press both :guilabel:`left` and :guilabel:`right` to open the signing options and press both :guilabel:`left` and :guilabel:`right` buttons again to sign the command.
+      
+   .. image:: imgs/mig-test/ledger-s-create-account-sign.png
+      :width: 300px
+
+   This securely stores the private key on the device.
+
+#. Once signed, the Insolar Wallet displays a wallet validation window.
+
+   .. image:: imgs/mig-test/one-more-thing.png
+      :width: 400px
+
+#. Wait for the validation to complete and see the congratulations message.
+
+   .. image:: imgs/mig-test/ledger-n-congrats.png
+      :width: 400px
+
+Once the Wallet is created, you can manage your XNS with it. Every login and XNS transfer operation requires the associated private key stored on the Ledger Nano, so the device must be connected to confirm these actions.
+
+.. _log-in-nano:
+
+Log in the connected Wallet and view balance
+--------------------------------------------
+
+To log in the Insolar Wallet connected to your Ledger Nano, go through the following steps:
+
+#. In Google Chrome, open the `Insolar Wallet <https://wallet.insolar.io>`_ and click :guilabel:`LOG IN`.
+#. In the **Log in** panel, click the :guilabel:`Hardware` tab.
+
+   .. image:: imgs/mig-test/login-hw.png
+      :width: 400px
+
+#. Make sure your Ledger Nano is connected, unlocked, and the Insolar application is launched on it.
+#. Specify the key number you chose upon :ref:`wallet creation <enter_key_number>` and click :guilabel:`CONNECT TO LEDGER NANO S`.
+
+   .. image:: imgs/mig-test/enter-key-number.png
+      :width: 400px
+
+Insolar Wallet recognizes the launched application on the device and automatically logs into the wallet. Once logged in, you can see your balance on the :guilabel:`Dashboard` tab.
```

Comment:
Insolar Wallet recognizes the launched application on the device and automatically logs **in to** the wallet. Once logged in, you can see your balance on the :guilabel:`Dashboard` tab.